app/mailers/user_mailer.rb

```diff
@@ -17,19 +17,25 @@ def claim_submitted_notification(user, claim)
     notify_email to: user.email,
                  subject: t(".subject"),
                  body: t(".body",
+                         user_name: user.first_name,
+                         organisation_name: claim.school.name,
                          reference: claim.reference,
                          amount: claim.amount.format(symbol: true, decimal_mark: ".", no_cents: false),
+                         sign_in_url:,
                          link_to_claim:)
   end
 
-  def claim_created_support_notification(claim, email)
+  def claim_created_support_notification(claim, email, user)
```

Comment:
This change also makes me think, why do we need both `email` and `user`? Is `email` ever different from `user.email`?